app/presenters/nav_tab/link_collection.rb

```diff
@@ -27,15 +27,25 @@ def self.tab_methods
   end
 
   def admin
-    default + admin_only
+    use_button + admin_only
```

Comment:
Since `use_button` is now returning just a single element, you can't `+` it. We're getting a bunch of test failures.